packages/transform/jest.config.mjs

```diff
@@ -0,0 +1,16 @@
+/** @type {import('@jest/types').Config.InitialOptions} */
```

Comment:
We now use `vitest` for testing, please refer to other packages in the workspace for examples.

Reply:
Ah makes sense, when I duplicated the files from `AutoEnsureMiddleware` to get started with it. I didn't touch anything, however I will copy those files again now with the updates.